{"uid":"8ca8b5f0837f810d","name":"test_delete_user_authorized_as_another_user","fullName":"tests.test_user_delete.TestUserDelete#test_delete_user_authorized_as_another_user","historyId":"f921dd3d5d139c34912ef23ea50982dc","time":{"start":1715836103234,"stop":1715836107976,"duration":4742},"description":"This test verifies that one user cannot delete another user","descriptionHtml":"<p>This test verifies that one user cannot delete another user</p>\n","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"This test verifies that one user cannot delete another user","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","steps":[{"name":"POST request to URL '/user/'","time":{"start":1715836103234,"stop":1715836104197,"duration":963},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"POST request to URL '/user/'","time":{"start":1715836104197,"stop":1715836105164,"duration":967},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"POST request to URL '/user/login'","time":{"start":1715836105165,"stop":1715836105631,"duration":466},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"DELETE request to URL '/user/96968'","time":{"start":1715836105631,"stop":1715836106707,"duration":1076},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"GET request to URL '/user/96968'","time":{"start":1715836106707,"stop":1715836107972,"duration":1265},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"User Deletion Tests"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_user_delete"},{"name":"subSuite","value":"TestUserDelete"},{"name":"host","value":"fv-az1530-303"},{"name":"thread","value":"2277-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_user_delete"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":155,"passed":66,"unknown":0,"total":221},"items":[{"uid":"3915fb2de3ce272f","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/222//#testresult/3915fb2de3ce272f","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715749662991,"stop":1715749667621,"duration":4630}},{"uid":"41aeb623311bd012","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/221//#testresult/41aeb623311bd012","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715663318337,"stop":1715663323123,"duration":4786}},{"uid":"5f1dee21c147a642","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/220//#testresult/5f1dee21c147a642","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715576930325,"stop":1715576934998,"duration":4673}},{"uid":"cf7f03ad16eb6e81","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/219//#testresult/cf7f03ad16eb6e81","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715490440440,"stop":1715490446206,"duration":5766}},{"uid":"707d6e1d1d5f5b60","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/218//#testresult/707d6e1d1d5f5b60","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715404026605,"stop":1715404031247,"duration":4642}},{"uid":"e7deb8e444ce54c1","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/217//#testresult/e7deb8e444ce54c1","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715317665794,"stop":1715317671478,"duration":5684}},{"uid":"206c9a35892dc6d4","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/216//#testresult/206c9a35892dc6d4","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715231329819,"stop":1715231334486,"duration":4667}},{"uid":"beeefe612d0cb7a5","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/215//#testresult/beeefe612d0cb7a5","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715144855777,"stop":1715144860766,"duration":4989}},{"uid":"6e6e5973ac996cef","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/214//#testresult/6e6e5973ac996cef","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1715058553153,"stop":1715058557713,"duration":4560}},{"uid":"92cca09b8c6165cb","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/213//#testresult/92cca09b8c6165cb","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nrequests.exceptions.SSLError: HTTPSConnectionPool(host='playground.learnqa.ru', port=443): Max retries exceeded with url: /api/user/ (Caused by SSLError(SSLError(1, '[SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1006)')))","time":{"start":1714972069316,"stop":1714972069833,"duration":517}},{"uid":"ef782ed7d08b31c0","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/212//#testresult/ef782ed7d08b31c0","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714885649097,"stop":1714885653635,"duration":4538}},{"uid":"5d54f8f5717dec52","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/211//#testresult/5d54f8f5717dec52","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714799217942,"stop":1714799222839,"duration":4897}},{"uid":"9b8d74bad7cc6417","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/210//#testresult/9b8d74bad7cc6417","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714712878014,"stop":1714712883229,"duration":5215}},{"uid":"671c15681e53fa3b","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/209//#testresult/671c15681e53fa3b","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714626551942,"stop":1714626556753,"duration":4811}},{"uid":"5a9427080c693f48","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/208//#testresult/5a9427080c693f48","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714540048098,"stop":1714540052783,"duration":4685}},{"uid":"86d1adc5f44ec35b","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/207//#testresult/86d1adc5f44ec35b","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714453646481,"stop":1714453651469,"duration":4988}},{"uid":"b5e51caf6f927312","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/206//#testresult/b5e51caf6f927312","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714367255263,"stop":1714367260228,"duration":4965}},{"uid":"46ebd2c56b4bb1e1","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/205//#testresult/46ebd2c56b4bb1e1","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714280888128,"stop":1714280892684,"duration":4556}},{"uid":"5ab7c2cf62332fe2","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/204//#testresult/5ab7c2cf62332fe2","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714194457410,"stop":1714194463328,"duration":5918}},{"uid":"5d7fc7364d3d1109","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/203//#testresult/5d7fc7364d3d1109","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1714108031368,"stop":1714108036091,"duration":4723}}]},"tags":["@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"]},"source":"8ca8b5f0837f810d.json","parameterValues":[]}